packages/core/content-releases/admin/src/components/ReleaseModal.tsx

```diff
@@ -99,3 +242,50 @@ export const ReleaseModal = ({
     </ModalLayout>
   );
 };
+
+/**
+ * Timezone hook - Generates the list of timezones and user's current timezone
+ */
+
+interface ITimezoneOption {
+  offset: string;
+  value: string;
+}
+
+const useTimezone = () => {
+  const timezoneList: ITimezoneOption[] = Intl.supportedValuesOf('timeZone').map((timezone) => {
+    /**
+     * This will be in the format GMT${OFFSET} where offset could be
+     * nothing, a four digit string e.g. +05:00 or -08:00
+     */
+    const offsetPart = new Intl.DateTimeFormat('en', {
+      timeZone: timezone,
+      timeZoneName: 'longOffset',
+    })
+      .formatToParts()
```

Comment:
Here is an interesting case, if you use `.formatToParts()` like that it would use the today's date to get the timezone offset. That means that for me is going to show "UTC+01:00 Europe/Madrid" 

But what if I selecting a date for July? Above you are using `.zonedTimeToUtc` to parse the date to UTC0 and it handles this case, I mean, considering I'm using Europe/Madrid as timezone and I select a date and hour in July, it would parse that date into UTC subtracting -2 to the time. But on my Timezone input I would see "UTC+01:00 Europe/Madrid", maybe this could cause confusion?

If you pass the selected date to `.formatToParts()` it would give you the offset based on that, so if I select a date on July in my dropdown I would see "UTC+02:00 Europe/Madrid"

wdyt? 

cc @joshuaellis

Reply:
Indeed, thats very important param I missed.